structure/0062-quilt-community-collab.md

```diff
@@ -0,0 +1,125 @@
+
+## Roles inside the server
+
+### Keyholder
+
+The keyholder for this space is the same as for the Quilt Community Team. This role is described in the [RFC 0007 # Keyholders](https://github.com/QuiltMC/rfcs/blob/master/structure/0007-community-team.md#keyholders).
+
+### Managers
+
+Managers are responsible for keeping the collab program up and running.
+
+- Maintain and organise the associated Discord server
+- Actioning changes to the collab-related spaces
+- Organise polls and votes as required by formal processes
+- Help resolve conflicts between communities
+- Stay in touch with any community part of the program
+
+Community Collab Managers are the same as Community Managers of the Quilt Community Team.
+
+### Collaborators
+
+Collaborators are moderators of one or more communities in the program. They have access to the server to the moderation part of the server, such as general moderator chats and case threads. Access is also given to the event category.
+
+### Event Planners
+
+Event Planners are members of the staff team of one or more communities in the program, who focuses on event organisation. They have access to the event part of the server.
+
+### Project Roles
+
+Other roles may be given surrounding the different projects happening inside the program. Those roles are informal and will depend on each project.
+
+### Invitees
+
+Invitees are temporary users, with no access to the rest of the server. They only have access to temporary channels required by the process they are visiting for.
+
+## Voting process
+
+If required, a vote among the Community Collab group can be called by a Manager. A poll and an associated thread are created in an internal channel designated for that purpose, including as many details as possible.
+
+Each community gets a single vote representing its member. They are free to write their own process for determining their vote, but it must follow those rules:
+
+- This process must be democratic and give a voice to any community member who is able to join the Community Collab server under the Collaborator role.
+- Each eligible voter must have single, equal voting power.
+- The process can remove the voting power under some inactivity threshold.
+
+Once a community has decided their vote, one member can report their vote in the thread, mentioning the community name, and the option being voted for, with corresponding proof (such as a screenshot of an internal poll).
+
+Each community also has the option of skipping its vote if none of its members has an opinion on the poll. Such instances should be reported the same way.
+
+Polls must include a closing date of at least ten days. If required, the closing date can be pushed away by a maximum of a month, upon the informal agreement with the majority of the other communities. During that period, any Collaborator can use their veto right (see below).
```

Comment:
Overall this process seems way too lengthy and involved, especially for things that in the end might have barely any effect, such as adding another community to the program. We already know what happened with how complex the CT voting system is, and this is... even worse, imo

Reply:
The voting process itself takes 10 days under normal circumstances. We need to provide each community with a decently long time to get a poll set up, vote and report the result. This is only three days longer than the CT process, because of that overhead.